src/components/vr-stereo-toggle/vr-stereo-toggle.js

```diff
@@ -20,6 +20,9 @@ const mapStateToProps = state => ({
   config: state.config.components.vrStereo
 });
 
+// TODO Sakal - Oren, the name in the constructor is incorrect (should have been VrStereoToggleControl)
```

Comment:
vr stereo toggle - change name to only vr-stereo to match file name and class name conventions and fix naming as described in Cast TODO comment for backward compatibility